app/controllers/admin/edition_unpublishing_controller.rb

```diff
@@ -4,6 +4,11 @@ class Admin::EditionUnpublishingController < Admin::BaseController
 
   def update
     if @unpublishing.update_attributes(explanation: params[:unpublishing][:explanation])
+      if withdrawing?
+        Whitehall::PublishingApi.publish_withdrawal_async(@unpublishing.content_id, @unpublishing.explanation)
```

Comment:
I think it's the `Document#content_id` you want here not the `Unpublishing`. We should probably make that more explicit in the `Whitehall::PublishingApi` code.